docs/quickstarts/nextjs.mdx

```diff
@@ -177,29 +206,30 @@ export default function Home() {
 
 </CodeBlockTabs>
 
-### Sign out of your application
-
-Visit your Next.js application at [`localhost:3000`](http://localhost:3000), click on your avatar, and select **Sign out**. You are now back where you started: a sign-in form.
+Then, visit your app's homepage at [`localhost:3000`](http://localhost:3000) to see the `<UserButton />` in action. It should show your avatar from the account you signed in with.
 
-### Deploy your application
+### Sign out
 
-You're ready to [deploy your app to production](/docs/deployments/overview) and welcome new users!
+Finally, use the `<UserButton />` by selecting your avatar, and select **Sign out**. You should be redirected to the url you specified with `afterSignOutUrl`.
 
 </Steps>
 
-## You're authenticated!
-
-Congratulations! Your app is now only available to authenticated users. But this is just the first step.
+## Next steps
 
-If you would like a repository that demonstrates many of the features Clerk has to offer, such as user, session, and organization management, check out one of Clerk's demo repositories:
+To see an example application that demonstrates user authentication, and session and organization management, explore one of Clerk's Next.js demo repos:
 
 - [Clerk + Next.js App Router Demo](https://github.com/clerk/clerk-nextjs-demo-app-router)
 - [Clerk + Next.js Pages Router Demo](https://github.com/clerk/clerk-nextjs-demo-pages-router)
 
-## Next steps
+## More resources
 
 <div className="container mx-auto my-4">
   <div className="grid grid-cols-1 gap-6 md:grid-cols-2">
+
+    <Cards title="Deploy to Production" description="Learn how to deploy your Clerk app to production." link="/docs/deployments/overview" cta="Learn More" />
+
+    <Cards title="Deploy to Vercel" description="Learn how to deploy your Clerk app to production on Vercel." link="/docs/deployments/deploy-to-vercel" cta="Learn More" />
+

```

Comment:
I'd even argue these would be the last steps in the line up. I feel that users will want to understand things like how to build a custom sign in page, how to read user data, how to use hooks, etc. before they would care about deploying to production or to vercel. 

what do you think?

Reply:
That makes sense!